Celeste.Mod.mm/Mod/Helpers/ModUpdaterHelper.cs

```diff
@@ -191,8 +191,14 @@ public static void TryDelete(string path) {
         /// This should point to a running instance of https://github.com/maddie480/EverestUpdateCheckerServer.
         /// </summary>
         private static string getModUpdaterDatabaseUrl(string database) {
+            if (CoreModule.Settings.UseAPIMirror) {
+                Logger.Info("ModUpdaterHelper", $"Using mirror for {database} database");
+                if (database == "modupdater") return "https://everestapi.github.io/updatermirror/everest_update.yaml";
+                if (database == "modgraph") return "https://everestapi.github.io/updatermirror/mod_dependency_graph.yaml";
+            }
```

Comment:
Consider moving the log inside of both if statements so that it only happens if it is actually going to return a mirror url.
```suggestion
            if (CoreModule.Settings.UseAPIMirror && (database == "modupdater" || database == "modgraph")) {
                Logger.Info("ModUpdaterHelper", $"Using mirror for {database} database");
                return $"https://everestapi.github.io/{database}/everest_update.yaml";
            }
```